examples/client-rendering/src/routes/__generated__/Route__Root__Todos__ByStatusDecoded__Child_route.res

```diff
@@ -165,4 +175,28 @@ external makeRenderer: (
   ~prepare: Internal.prepareProps => 'prepared,
   ~prepareCode: Internal.prepareProps => array<RelayRouter.Types.preloadAsset>=?,
   ~render: Internal.renderProps<'prepared> => React.element,
-) => Internal.renderers<'prepared> = ""
\ No newline at end of file
+) => Internal.renderers<'prepared> = ""
+
+
+@live 
+let parseRoute = (route: string, ~exact=false): option<(
+  pathParams,
+  queryParams,
+)> => {
+  switch route->String.split("?") {
```

Comment:
Might make sense for us to use `URL` here. Or I guess it doesn't matter much, splitting on `?` will always work anyway.

Reply:
Hehe I precisely thought of using URL too but the issue is the routes we generate are not valid URL since we don't have the domain name, so splitting always works.